### DISCUSSION TITLE
Is there anyway to export the document to PC?

### DISCUSSION THREAD

**Discussion by PurpleSweetPotato27** [keiwando/toonsquid#699] [General]
it’s a great app to make animation ,
and is there any way to keep the regional document to pc or 
just the editable file transform to AfterEffect? 

(Animation Student , non-native English user QAQ)

**Comment by budaloops** [comment]
HimPutpleSweetPotato27. It does not look like there is a way to transfer an editable file to After Effects at the moment.

**Reply by PurpleSweetPotato27** [reply]
T-T thank U！omg hope could  be editable in the future

**Comment by andydjames** [comment]
You can export your ToonSquid animation as an image sequence in the following formats.
PNG
JPEG (no transparency)
TIFF
BMP (no transparency)
TGA

You can do an image sequence export (say to png with transparency) of the project separating out the layers.
Hide all layers, except the one you wish to export, then export image sequence. It will just export the visible layer.
Repeat for each layer. 
You can then import each layer into AfterEffects as an image sequence to keep them separated. You won't retain any keyframing info, of course.

**Reply by PurpleSweetPotato27** [reply]
Oh!!! Thank you for the suggestion, its a great way to get single layer out,
 but I wanna a whole file could trans to PC, seems impossible now, hope could be true someday (T_T)
